internal/cloudapi/v2/errors.go

```diff
@@ -219,7 +220,8 @@ func (s *Server) HTTPErrorHandler(echoError error, c echo.Context) {
 			apiErr := APIError(code, sec, c)
 
 			if sec.httpStatus == http.StatusInternalServerError {
-				c.Logger().Errorf("Internal server error. Code: %s, OperationId: %s", apiErr.Code, apiErr.OperationId)
+				logrus.Errorf("Internal server error. Code: %s, OperationId: %s, internalError: %v", apiErr.Code, apiErr.OperationId,
+					echoError.(*echo.HTTPError).Internal)
```

Comment:
hm this type assertion will fail in of ErrorNotHTTPError no?

Reply:
I cannot be completely sure so I added a check